Bottom: 06e58fa3ef0c4bd7641eb90e820b82f024c9d5d0
Top:    00e3fcad61ad2ade2bb5636052d622980acad194
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-01 01:44:37 +0100

Refresh of polygon1-c-temporarily-disable

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 84b30ea..9157eb4 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -600,7 +600,6 @@ adjust_tree (rtree_t * tree, struct seg *s)
   return 0;
 }
 
-
 #if 0  /* TURNS OUT DISABLING THIS ACTUALLY (VERY MARGINALLY) SPEEDS THINGS UP! */
 /*
  * seg_in_region()
